inference-engine/tests/functional/inference_engine/ngraph_reader/gather_tests.cpp

```diff
@@ -8,55 +8,60 @@ TEST_F(NGraphReaderTests, ReadGatherNetwork) {
     std::string model = R"V0G0N(
 <net name="saved_model" version="10">
 	<layers>
-		<layer id="0" name="input_a" type="Parameter" version="opset1">
-			<data shape="1,3,4" element_type="f32"/>
+		<layer id="0" name="data" type="Parameter" version="opset1">
+			<data shape="2,3,4" element_type="f32"/>
 			<output>
 				<port id="0" precision="FP32">
-					<dim>1</dim>
+					<dim>2</dim>
 					<dim>3</dim>
 					<dim>4</dim>
 				</port>
 			</output>
 		</layer>
-		<layer id="1" name="input_b" type="Parameter" version="opset1">
-			<data shape="1" element_type="i32"/>
+		<layer id="1" name="indices" type="Parameter" version="opset1">
+			<data shape="8,16" element_type="i32"/>
 			<output>
 				<port id="0" precision="I32">
-					<dim>1</dim>
+					<dim>8</dim>
+					<dim>16</dim>
 				</port>
 			</output>
 		</layer>
-		<layer id="2" name="PartitionedCall/functional_1/tf_op_layer_GatherV2/GatherV2/Cast_292_const" type="Const" version="opset1">
+		<layer id="2" name="GatherV2/axis" type="Const" version="opset1">
 			<data offset="0" size="8" shape="" element_type="i64"/>
 			<output>
 				<port id="1" precision="I64"/>
 			</output>
 		</layer>
-		<layer id="3" name="PartitionedCall/functional_1/tf_op_layer_GatherV2/GatherV2" type="Gather" version="opset1">
+		<layer id="3" name="Gather" type="Gather" version="opset7">
+            <data batch_dims="0"/>
```

Comment:
Please, keep the original test for Gather1 as well. We should not modify existing tests. We should introduce new ones!

Reply:
returned back the old test as well